src/caustic/parametrized.py

```diff
@@ -401,7 +400,21 @@ def add_params(p: Parametrized, dot):
 
         return dot
 
+class Simulator(Parametrized):
```

Comment:
This should be in a separate file called simulator.py. Even though it is a very small wrapper, we might add stuff in the future.

Reply:
Ok, now its in a separate file